Wikipedia/Code/UIColor+WMFStyle.h

```diff
@@ -67,4 +67,20 @@
 
 + (instancetype)wmf_settingsBackgroundColor;
 
+///
+/// @name Derivative Colors
+///
+
+/**
+ *  @return A dimmed copy of the receiver.
+ *
+ *  @see -wmf_colorByScalingComponents:
+ */
+- (instancetype)wmf_colorByApplyingDim;
```

Comment:
IMO that's backwards. this method says exactly what it does—it's up to the application to decide when it would be used. so far, it's only used to dim a color when a label is highlighted